Eatery/Controllers/Eateries/PillViewController.swift

```diff
@@ -116,11 +116,15 @@ class PillViewController: UIViewController {
     }
 
     @objc private func pillSelectionDidChange() {
+        let payload : Payload!
```

Comment:
In this case, Swift is smart enough to determine when the variable is initialized. You can remove the implicity unwrapped optional. 

`let payload: Payload`

And initialize it in each branch of the if